docs/pages/management/guides/joining-nodes-azure.mdx

```diff
@@ -0,0 +1,232 @@
+the **Access control (IAM)** tab. In the row of buttons at the top of the
+**Access control (IAM)** panel, click **Add > Add role assignment**.
+
+Within the **Add role assignment** screen, click **teleport-read-vm**.
+
+![Add a role
+assignment](../../../img/azure/add-role-assignment.png)
+
+Scroll to the bottom of the screen and click **Next**.
+
+Within the **Members** tab, in the **Assign access to** field, choose **Managed
+identity**. Click **Select members**.
+
+On the right sidebar, find the **Managed identity** dropdown menu and select
+**User-assigned managed identity**. Choose the `teleport-azure` identity you
+created earlier.
+
+![Select managed
+identities](../../../img/application-access/azure/select-managed-identities.png)
+
+Click **Select**, then **Review + assign**.
+
+Verify that your **Role** is `teleport-read-vm`, the **Scope** matches your
+chosen resource group, and the **Members** field includes the `teleport-azure`
+managed identity you created earlier.
+
+Click **Review + assign** again.
+
+### Attach an identity to your Azure VM
+
+In the [Virtual machines
+view](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines)
+of Azure Portal, click on the name of the VM you are using to host the Teleport
+Application Service. 
+
+On the right side panel, click the **Identity** tab, then within the
+**Identity** view, click the **User assigned** tab. Click **+Add**, then select
+the `teleport-azure` identity. Click **Add**.
+
+![Add an identity to a
+VM](../../../img/application-access/azure/vm-identity.png)
+
+Navigate back to **Identity** tab in the page for your Azure VM. You should see
+the new identity listed in the **User assigned** sub-tab:
+
+![Verifying that you added the
+identity](../../../img/application-access/azure/verify-id.png)
+
+<Admonition
+  type="warning"
+  title="IMPORTANT"
+>
+As part of authentication, Teleport will be able to do anything allowed by the
+identity. If you use a managed identity other than the one created in this
+guide, we strongly recommend limiting its permissions and scope. 
+</Admonition>
+
+## Step 2/4. Create the Azure Node joining token
+Under the hood, Nodes will prove that they are running in your Azure subscription
+by sending a signed attested data document and access token to the Teleport Auth
+Service. The Node's identity must match an allow rule configured in your Azure
+Node joining token.
+
+Create the following `token.yaml` with an `allow` rule specifying your Azure
+subscription and the resource group that your Node's identity must match.
+
+```yaml
+# token.yaml
+kind: token
+version: v2
+metadata:
+  # the token name is not a secret because instances must prove that they are
+  # running in your Azure subscription to use this token
+  name: azure-token
+  # set a long expiry time, the default for tokens is only 30 minutes
+  expires: "3000-01-01T00:00:00Z"
```

Comment:
Is this necessary? I think @strideynet may have made some changes to make it optional.